### DISCUSSION TITLE
Twarc Citation

### DISCUSSION THREAD

**Discussion by Terela62** [DocNow/twarc#624] [General]
Can a Twarc citation be added to the Github repository so I can reference this in a paper?

**Comment by edsu** [comment]
Thanks for citing! There is a badge on the README that contains the DOI:

[![DOI](https://zenodo.org/badge/7605723.svg)](https://zenodo.org/badge/latestdoi/7605723)

Does that help?

**Reply by Terela62** [reply]
It does thank you!
